compute/cker/include/cker/operation/Conv.h

```diff
@@ -61,11 +61,13 @@ class Conv
   }
 
   void prepare(const Shape &filter_shape, const float *filter_data, PaddingType padding_type,
-               bool &is_replaced_weights)
+               bool &is_replaced_weights, uint32_t dilationWidthFactor,
+               uint32_t dilationHeightFactor)
   {
     if (!_prepared)
     {
-      if (usableMultiThreaded(padding_type))
+      if (usableMultiThreaded(padding_type) && dilationWidthFactor == 1 &&
+          dilationHeightFactor == 1)
```

Comment:
@YiHyunJin Could you please move additional conditions of dilation into `usableMultiThreaded`?